### DISCUSSION TITLE
Credenziali non valide

### DISCUSSION THREAD

**Discussion by Montalbano1972** [indomus/forum#961] [HUB: Home Assistant]
buongiorno,
ho installato HA su windows (con Container) e ho integrato con successo uno shelly.
Oggi, ho provato a loggarmi ma sembra che, o password, oppure User non siano validi
come posso recuperare le credenziali?

grazie, saluti
Alessandro

**Comment by snakuzzo** [comment]
https://www.home-assistant.io/docs/locked_out/
